android/rnkiwimobile/src/main/java/com/kiwi/rnkiwimobile/account/RNAccountSettingsActivity.kt

```diff
@@ -0,0 +1,9 @@
+package com.kiwi.rnkiwimobile.account
+
+import com.kiwi.rnkiwimobile.RNKiwiActivity
+
+abstract class RNAccountSettingsActivity(initialProperties: RNAccountInitialProperties) :RNKiwiActivity(RNAccountModule.getInitialProperties(initialProperties)) {
```

Comment:
I don't know how much do you watch code formatting in this project but the semicolon between new class and base class is not formatted as in the rest of the project.

Reply:
Now that you mentioned it, enough to trigger my OCD  🙈 

Updated now 😊